### DISCUSSION TITLE
Is it possible to disable Anthropic token caching by default?

### DISCUSSION THREAD

**Discussion by ThomasBaruzier** [danny-avila/LibreChat#4517] [Q&A]
Hello,

Kudos for the new rounded CSS, I like it.

I only do 1 prompt 1 answer as Sonnet can answer my questions first try most of the time. Can I disable prompt caching by default? In the modelSpecs, by preference?

**Comment by jtomanik** [comment]
if the switch in parameters UI is not enough for you, you can always create a preset (in UI) for Anthropic with caching disabled.
